pom.xml

```diff
@@ -136,6 +136,7 @@
         <module>webtau-dist</module>
         <module>webtau-cli-testing</module>
         <module>webtau-docs</module>
+        <module>webtau-coverage</module>
```

Comment:
let's rename to `webtau-jacoco-coverage` to avoid confusion with potential data coverage module we may introduce